database.go

```diff
@@ -78,15 +78,36 @@ func (e *Env) GetLatest() ([]Task, error) {
 	}
 	defer rows.Close()
 
-	reports, err := ParseRowsTasks(rows)
+	tasks, err := ParseRowsTasks(rows)
 	if err != nil {
 		return []Task{}, fmt.Errorf("failed to parse rows: %s", err)
 	}
 
-	return reports, nil
+	return tasks, nil
 
 }
 
+// func (e *Env) GetTaskById() ([]Task, error) {
```

Comment:
We're not at home to commented-out code here at BIT! I'm sure you meant to hit 'delete' instead, didn't you? 😁